COMPATIBILITY.md

```diff
@@ -0,0 +1,30 @@
+# Support and Compatibility for fabric-contract-api-go
+
+GitHub is used for code base management, issues should be reported in the [FABCAG](https://jira.hyperledger.org/projects/FABCAG/issues) component in JIRA.
+
+## Summary of Compatibility
+
+This table shows the summary of the compatibility of the package at version 2.0, together with the Go version it requires and the Fabric Peer version it can communicate with.
+
+|                       | Peer Connectivity v2.0 | Go 1.13 |
+| --------------------- | ---------------------- | ------- |
+| Packages **v2.0**     | Yes                    | Yes     |
+
```

Comment:
Do we explicitly want to mention we're not supporting v1.4? I'm wondering whether it's worth getting Lesley-Ann's thoughts?

Reply:
It probably is worth asking her to cast an eye over it. I have now updated the table so it may make it clearer